multigeiger/multigeiger.ino

```diff
@@ -384,8 +375,17 @@ char *buildSSID() {
 // ******* SETUP *******
 //====================================================================================================================================
 void setup() {
-  // OLED-Display
+  // init OLED-Display
   u8x8.begin();
+  // On the WIFI CPU u8x8.begin() cleared the display.
+  #if CPU == STICK
+  // But not on the STICK CPU, so we do it here.
+  // Display is only 4 lines by 8 characters; lines counting from 2 to 5
+  u8x8.setFont(u8x8_font_5x8_f);        // use really small font
+  for (int i = 2; i < 6; i++) {
+    u8x8.drawString(0, i, "        ");  // clear all 4 lines
+  }
+  #endif
```

Comment:
if u8x8.begin() clears the WIFI display, but not the STICK display, doing both here should be structurally better, right?